NOTICE

```diff
@@ -0,0 +1,9 @@
+
+Apache Uniffle (incubating)
```

Comment:
First character should be upper case.

Reply:
No need for (word)
https://github.com/apache/incubator-mxnet/blob/master/NOTICE
https://github.com/apache/incubator-devlake/blob/main/NOTICE

Reply:
Consistent with DISCLAIMER.

Reply:
Both are updated